fendermint/testing/materializer/golden/manifest/toml/manifest.toml

```diff
@@ -0,0 +1,133 @@
+[accounts.My2Bn2XY]
+
+[accounts.OK5OS6]
+
+[accounts.Z66]
+
+[accounts.wk8j]
+
+[accounts.z_MP]
+
+[rootnet.External.deployment.Existing]
```

Comment:
Is there a way to get rid of this uppercase and lowercase concoction?

Reply:
We can move them inside the struct, in which case it would still be uppercase but wouldn't be a key: https://serde.rs/enum-representations.html

So instead of this
```json
"rootnet": { 
  "External": {
    "deployment": {
      "Existing": {
        "gateway": "...", 
        "registry": "..."
      }
    }
  }
}
```
it would be like:
```json
"rootnet": { 
    "type": "External",
    "deployment": {
        "type": "Existing",
        "gateway": "...", 
        "registry": "..."
    }
}
```

I'm 90% sure there is also a way to apply some [attribute](https://serde.rs/container-attrs.html) to `lowercase`, but I'm not sure. If it would be internally tagged, I think that wouldn't be necessary. 

Would you prefer internal, or external lowercase?

Reply:
Switched to internal.